components/03-forms/10-misc/01-buttons/buttons.hbs

```diff
@@ -1,7 +1,22 @@
 <h2 style="margin-bottom: 24px">Primaire button / Call to action</h2>
+
+<div class="row">
+  <div class="col-md-9">
+    <div class="dso-rich-content">
+      <label>Gebruik:</label>
```

Comment:
Ik vind `<label>` eigenlijk best gek hier, maar het komt al vaker voor. Dus, de semantiek van dit soort toelichtende pagina's moeten we een andere keer nog maar eens onder de loep nemen.